app/models/ingredient.rb

```diff
@@ -1,6 +1,7 @@
 class Ingredient < ApplicationRecord
+  validates :name, :unit_type, presence: true, length: {minimum: 2}
+  validates_uniqueness_of :name, message: 'that ingredient already exist'
```

Comment:
Nobody really knows what "than ingredient" really means. I think that default message which is something like "is no unique" is just enough.